src/ray/util/process.h

```diff
@@ -0,0 +1,131 @@
+#ifndef RAY_UTIL_PROCESS_H
+#define RAY_UTIL_PROCESS_H
+
+#ifdef __linux__
+#include <fcntl.h>     // O_RDONLY
```

Comment:
Let's remove the comments ``// O_RDONLY`` etc from all the includes. They *will* get out of sync (and we don't use comments like this anywhere else in the codebase).